databook/_toc.yml

```diff
@@ -11,79 +11,95 @@ chapters:
+  - file: physiology/ophys/ophys-background                    # Also a placeholder that needs actual content
     sections:
-    - file: visual-behavior/VB-BehaviorSessionData
-      title: Behavior Session Data
-  - file: visual-behavior/VB-Ophys
-    title: Visual Behavior Ophys
+    - file: physiology/ophys/visual-coding/vc2p-background
+      sections:
+      - file: physiology/ophys/visual-coding/vc2p-dataset
+      - file: physiology/ophys/visual-coding/vc2p-session-data
+      - file: physiology/ophys/visual-coding/vc2p-stimuli
+      - file: physiology/ophys/visual-coding/vc2p-evoked-response
+      - file: physiology/ophys/visual-coding/vc2p-cross-session-data
+      - file: physiology/ophys/visual-coding/vc2p-analysis
+    - file: physiology/ophys/visual-behavior/VB-Ophys
+      title: Visual Behavior Ophys
+      sections:
+      - file: physiology/ophys/visual-behavior/VBO-Dataset
+        title: Visual Behavior Ophys Dataset
+      - file: physiology/ophys/visual-behavior/VBO-ExperimentData
+        title: Ophys Experiment Data
+      - file: physiology/ophys/visual-behavior/VBO-Tutorial-Compare_trial_types
+        title: Tutorial Comparing Trial Types
+      - file: physiology/ophys/visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
+        title: Tutorial Downloading Data
+    - file: physiology/ophys/learning/learning                    # Also a placeholder that needs actual content
+    - file: physiology/ophys/inhibitory-balance/inhibitory-balance                    # Also a placeholder that needs actual content
+  - file: physiology/ephys/ephys-background                    # Also a placeholder that needs actual content
     sections:
-    - file: visual-behavior/VBO-Dataset
-      title: Visual Behavior Ophys Dataset
-    - file: visual-behavior/VBO-ExperimentData
-      title: Ophys Experiment Data
-    - file: visual-behavior/VBO-Tutorial-Compare_trial_types
-      title: Tutorial Comparing Trial Types
-    - file: visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
-      title: Tutorial Downloading Data
-  - file: visual-behavior/VB-Neuropixels
-    title: Visual Behavior Neuropixels
+    - file: physiology/ephys/visual-coding/vcnp
+      sections:
+      - file: physiology/ephys/visual-coding/vcnp-quickstart
+      - file: physiology/ephys/visual-coding/vcnp-data-access
+      - file: physiology/ephys/visual-coding/vcnp-units
+      - file: physiology/ephys/visual-coding/vcnp-receptive-fields
+      - file: physiology/ephys/visual-coding/vcnp-lfp
+      - file: physiology/ephys/visual-coding/vcnp-optotagging
+      - file: physiology/ephys/visual-coding/vcnp-session
+      - file: physiology/ephys/visual-coding/vcnp-cheatsheet
+      - file: physiology/ephys/visual-coding/vcnp-links
+    - file: physiology/ephys/visual-behavior/VB-Neuropixels
+      title: Visual Behavior Neuropixels
+      sections:
+      - file: physiology/visual-behavior/VBN-Dataset
```

Comment:
all these VBN pages need to be in the ephys sub-folder:
physiology/ephys/visual-behavior/VBN-xxx